test/e2e/support/world.js

```diff
@@ -0,0 +1,57 @@
+const { expect, assert } = require('chai');
+const config = require('./config');
+const { setWorldConstructor } = require('cucumber');
+const minimist = require('minimist');
+
+const argv = minimist(process.argv.slice(2));
+const { setDefaultTimeout } = require('cucumber');
+
+setDefaultTimeout(60 * 1000);
+
+function processRecursive(part) {
+    if (part in config.lookups) {
+        if (config.lookups[part].indexOf('|') === -1) {
+            return config.lookups[part];
+        }
+        return processSelector(config.lookups[part]);
+    }
+}
+
+function processSelector(selector) {
+    if (selector.search(/\s/ig) !== -1) {
+        throw new Error('selector cannot contain spaces');
+    }
+    return selector.split('|')
+        .map(level => level.split('.')
+            .map(part => processRecursive(part) || `[data-selector~="${part}"]`)
+            .join(''))
+        .join(' ');
+}
+
+const seconds = n => n * 1000;
```

Comment:
this function doesn't appear to be used, though it could be

Reply:
these values are not used world.js and timeout.js, as we used the same old files which we will be refactoring in our tidyup tasks